docs/sources/flow/reference/components/otelcol.connector.spanmetrics.md

```diff
@@ -0,0 +1,267 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.connector.spanmetrics/
+title: otelcol.connector.spanmetrics
+---
+
+# otelcol.connector.spanmetrics
+
+`otelcol.connector.spanmetrics` accepts span data from other `otelcol` components and 
+aggregates Request, Error and Duration (R.E.D) OpenTelemetry metrics from the spans:
+
+* **Request** counts are computed as the number of spans seen per unique set of dimensions, 
+including Errors. Multiple metrics can be aggregated if, for instance, a user wishes to 
+view call counts just on `service.name`` and `span.name``.
+
+* **Error** counts are computed from the Request counts which have an `Error` status code metric dimension.
+
+* **Duration** is computed from the difference between the span start and end times and inserted 
+into the relevant duration histogram time bucket for each unique set dimensions.
+
+> **NOTE**: `otelcol.connector.spanmetrics` is a wrapper over the upstream
+> OpenTelemetry Collector `spanmetrics` connector. Bug reports or feature requests
```

Comment:
Those are both great ideas! Thank you. We currently don't do either of those things for otelcol components. I opened a separate issue - #4917 - so that we can discuss those ideas and a few more. Hopefully we can update all components in bulk. I'd prefer not to do such updates in this PR just for this component, for consistency.